src/main/java/pl/sda/eventsagregator/domain/user/Role.java

```diff
@@ -0,0 +1,10 @@
+package pl.sda.eventsagregator.domain.user;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public enum Role {
+    A("admin"), O("organizer"), U("user");
```

Comment:
Please make full enum - names. ADMIN, ORGANIZER, USER instead of A, O, U -> will be easier to work with